code/_expts/slotnick_etal_2001/analyze/plots/de_CreateSlotnickFigure1.m

```diff
@@ -29,3 +29,7 @@
     xlabel('Hemisphere Injected');  % weird, right??
     ylabel(ax, 'Sum squared error');
     title(ax, sprintf('%s\n%s\n ', taskTitle, stimSet));
+    
+    yrule = ax.YAxis;
+    yrule.FontSize=15
```

Comment:
That changes the size of the labels, which I don't care about. I want to change the size of the YTickLabels and that seems to be the only way to do it. [You are right though, it was a feature introduced in 2015B ](https://www.mathworks.com/matlabcentral/answers/298117-independent-xticklabel-and-yticklabel-font-sizes)

Not really sure what the solution is. I can comment it out and put a note there.